src/shared/dissect-image.c

```diff
@@ -2089,7 +2089,7 @@ static int mount_partition(
                                 (void) fs_grow(node, -EBADF, p);
 
                         if (userns_fd >= 0) {
-                                r = remount_idmap_fd(STRV_MAKE(p), userns_fd);
+                                r = remount_idmap_fd(STRV_MAKE(p), userns_fd, 0);
```

Comment:
our coding style suggests to prefix arguments in function calls with a comment such as `/* extra_flags= */` if they otherwise appear entirely opaque on what they might be, i.e. whenever you hardcode numeric literals or NULL or so. hence here, prefix the last argument like that. (and everywhere else, where you pass just 0 or NULL or so to some argument)